src/include/utils/plancache.h

```diff
@@ -110,6 +110,8 @@ typedef struct CachedPlanSource
 	double		generic_cost;	/* cost of generic plan, or -1 if not known */
 	double		total_custom_cost;		/* total cost of custom plans so far */
 	int			num_custom_plans;		/* number of plans included in total */
+	bool		is_custom_cost_safe;	/* is it safe to compare costs of
+										 * generic and custom plan */
```

Comment:
Changing the name to `is_postgres_planner_used` will change variable semantics, which should lead to moving it's calculation logic closer to statements generation, e.g. somewhere [here](https://github.com/arenadata/gpdb/blob/d627e197a31c04d2049f32028a71f6507eb6ea79/src/backend/utils/cache/plancache.c#L945). As `BuildCachedPlan()` can be used for generating Generic plans, it will be better to name it like `is_postgres_planner_used_for_custom` or similar. 
I don't think we really need this.